sql/moz-fx-data-shared-prod/adjust_derived/adjust_derived_v1/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+---
+friendly_name: Data from external source Adjust
+description: >
+  This table represents data downloaded from Adjust
+owners:
+  - mhirose@mozilla.com
+labels:
+  incremental: false
+scheduling:
+  dag_name: bqetl_adjust
+  date_partition_parameter: null
```

Comment:
is your table expected to be partitioned by date?

Reply:
Yes, it will be one table partitioned by date